README.md

```diff
@@ -48,14 +48,13 @@ Moreover, having these extensions will help you to speed up the development proc
     npm install
     ```
 
-
 5.  **To start developing:**
 
     ```sh
     npm run dev
     ```
 
-1. Release to staging:
+6.  Release to staging:
```

Comment:
Why does this change to `6`

Reply:
I think readme was accidentally prettified somehow. Will revert the changes on the readme